src/main/plugins/org.dita.pdf2/cfg/fo/attrs/loc-attr.xsl

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0'?>
+
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:fo="http://www.w3.org/1999/XSL/Format"
+                xmlns:rx="http://www.renderx.com/XSL/Extensions"
+                version="2.0">
+
+    <xsl:attribute-set name="releaseManagementTable">
+    </xsl:attribute-set>
+
+    <xsl:attribute-set name="releaseManagementTable.bookRelease.row">
```

Comment:
Tables/rows/cells should use attribute sets from normal table styling, e.g. `use-attribute-sets="tbody.row"` in this case.